sources/packages/web/src/composables/useFormatters.ts

```diff
@@ -33,6 +33,13 @@ export function useFormatters() {
     return "";
   };
 
+  const timeOnlyString = (date: string | Date): string => {
+    if (date) {
+      return dayjs(date).format("HH:mm");

```

Comment:
are we not showing the timezone?

Reply:
as displayed in user timezone it is not explicitly mentioned.